### PR TITLE
Store entity is done

### PR DESCRIPTION
**Pull request by JavaGuru17**

### CODE REVIEW

File: src/main/java/uz/pdp/eufloria/domain/Store.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private LocalTime opens;
+    private LocalTime closes;
+    @OneToOne
+    private Address address;
+    @OneToMany
+    @ToString.Exclude
```
Comment (byte-11) [inline src/main/java/uz/pdp/eufloria/domain/Store.java]: remove @ToString.Exclude , or you need it ?

File: src/main/java/uz/pdp/eufloria/domain/Store.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import lombok.ToString;
+
+import java.time.LocalTime;
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
```
Comment (byte-11) [inline src/main/java/uz/pdp/eufloria/domain/Store.java]: Why you need @ToString annotation ?

Comment (JavaGuru17) [reply src/main/java/uz/pdp/eufloria/domain/Store.java]: I thought that after some time maybe it will be needful. But now I think it doesn't need

Comment (JavaGuru17) [reply src/main/java/uz/pdp/eufloria/domain/Store.java]: ToString is removed

**Review by byte-11 — APPROVED** [review]

**Pull request merged**